### PR TITLE
Add database table for events

### LINKED ISSUE #53: Task 1.1.3: Create a database schema for the events

### PR DESCRIPTION
**Pull request by gurnoor6**
This pull request resolves #53

### CODE REVIEW

**Review by DaveCSW — COMMENTED** [review]
reviewed

**Review by sudolucy — APPROVED** [review]
Schema looks good to me! Thanks!

File: sql/create_tables.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS events (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    venue VARCHAR(255) NOT NULL,
+    date DATE NOT NULL,
+    time TIME NOT NULL,
+    description TEXT,
+    tags VARCHAR(255),
+    imagepath VARCHAR(255),
```
Comment (anmoldtu) [inline sql/create_tables.sql:9]: We have to figure out how images will be handled. Maybe can look at it in later changes. Fine for now!

**Review by sudolucy — APPROVED** [review]

**Review by anmoldtu — APPROVED** [review]

**Review by DaveCSW — COMMENTED** [review]
good

**Pull request merged**